src/components/InvertedFlatList/CellRendererComponent/index.android.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../../styles/styles';
+
+const propTypes = {
+    /** Position index of the list item in a list view */
+    index: PropTypes.number.isRequired,
+};
+
+function CellRendererComponent(props) {
+    return (
+        <View
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...props}
+            style={[
+                styles.invert,
+                // To achieve absolute positioning and handle overflows for list items,
```

Comment:
NAB but I think out style is to have blank lines before comments (unless the comment starts at a new level of indentation)
```suggestion

                // To achieve absolute positioning and handle overflows for list items,
```

Reply:
changed comment style according to https://github.com/airbnb/javascript#comments--multiline